eurec4a_environment/nomenclature.py

```diff
@@ -0,0 +1,168 @@
+# TODO: update altitude to be called `alt` once JOANNE dataset is released
+# which uses this definition
+ALTITUDE = "height"
+RELATIVE_HUMIDITY = "rh"
+POTENTIAL_TEMPERATURE = "theta"
+PRESSURE = "p"
+
+# From CF-conventions: "specific" means per unit mass. Specific humidity is
+# the mass fraction of water vapor in (moist) air.
+SPECIFIC_HUMIDITY = "q"
+
+# From CF-conventions: Speed is the magnitude of velocity. Wind is defined as
+# a two-dimensional (horizontal) air velocity vector, with no vertical
+# component. (Vertical motion in the atmosphere has the standard name
+# upward_air_velocity.) The wind speed is the magnitude of the wind velocity
+WIND_SPEED = "wspd"
+
+# From CF-conventions: "Eastward" indicates a vector component which is
+# positive when directed eastward (negative westward). Wind is defined as a
+# two-dimensional (horizontal) air velocity vector, with no vertical component.
+# (Vertical motion in the atmosphere has the standard name
+# upward_air_velocity.)
+ZONAL_WIND = "u"
+
+# From CF-conventions: "Northward" indicates a vector component which is
+# positive when directed northward (negative southward). Wind is defined as a
+# two-dimensional (horizontal) air velocity vector, with no vertical component.
+# (Vertical motion in the atmosphere has the standard name
+# upward_air_velocity.)
+MERIDIONAL_WIND = "v"
+
+
+CF_STANDARD_NAMES = {
+    TEMPERATURE: "air_temperature",
+    ALTITUDE: "geopotential_height",
+    RELATIVE_HUMIDITY: "relative_humidity",
+    POTENTIAL_TEMPERATURE: "air_potential_temperature",
+    PRESSURE: "air_pressure",
+    SPECIFIC_HUMIDITY: "specific_humidity",
+    WIND_SPEED: "wind_speed",
+    ZONAL_WIND: "eastward_wind",
+    MERIDIONAL_WIND: "northward_wind",
+}
+
+
+def _get_calling_function_name():
+    curframe = inspect.currentframe()
+    calframe = inspect.getouterframes(curframe, 2)
+    return calframe[1][3]
+
+
+class FieldMissingException(Exception):
+    pass
+
+
+def get_field(ds, field_name, units=None):
+    """
+    Get field described by `field_name` in dataset `ds` and ensure it is in the
+    units defined by `units` (if `units` != None). The units definition is any
+    unit definition supported by
+    [UDUNITS](https://www.unidata.ucar.edu/software/udunits/)
+    """
+    if field_name in ds:
+        da = ds[field_name]
+    else:
+
+        calling_function_name = _get_calling_function_name()
+
+        if field_name not in CF_STANDARD_NAMES:
+            raise FieldMissingException(
+                f"Couldn't find the variable `{field_name}` in the provided dataset."
+                f" To use {calling_function_name} you need to provide a variable"
+                f" with the name `{field_name}`"
+            )
+        else:
+            standard_name = CF_STANDARD_NAMES[field_name]
+            matching_dataarrays = {}
+            vars_and_coords = list(ds.data_vars) + list(ds.coords)
+            for v in vars_and_coords:
+                print(v)
```

Comment:
I don't think we want the print statement

Reply:
Good catch, thanks :)